engine/src/main/java/org/destinationsol/game/drawables/RectSprite.java

```diff
@@ -136,9 +136,9 @@ public void draw(GameDrawer drawer, SolGame game) {
             y = (y - camPosition.y) / level.depth + camPosition.y;
         }
         if (isAdditive) {
-            drawer.drawAdditive(texture, textureSizeX, textureSizeY, originalX, originalY, x, y, angle, tint);
+            drawer.drawAdditive(getTexture(), textureSizeX, textureSizeY, originalX, originalY, x, y, angle, tint);
```

Comment:
Is there any particular reason to use getTexture() instead of texture here? Even though it's not a big deal, avoiding an extra function call that can easily be avoided (without any loss to readability) seems like a no-brainer to me.